README.md

```diff
@@ -115,7 +115,7 @@ As mentioned above, if `https://` is not specified, connection will be redirecte
 
 If you want to build your own image, instead of directly pull the pre-built one, clone this repository, and run `docker build` command, with build-arg `MAXMIND_LICENSE_KEY`, you need to provide your own MaxMind license key from https://www.maxmind.com/en/account, it's **free**.
 
-```sh
+```shell
```

Comment:
It's quote common to just use `sh` I think, and also, there are multiple places use `sh` in this file, so prefer to keep it just `sh`, would you mind to help revert this change? Thanks.

Reply:
Of course

Reply:
https://github.com/PeterDaveHello/ipinfo.tw/pull/9/commits/1b453fd6c9b474505a933d9baf117cdab82d700f

I would have changed them all to shell if the change was to go ahead.